protocol.go

```diff
@@ -0,0 +1,131 @@
+package tarantool
+
+import (
+	"context"
+	"fmt"
+)
+
+type ProtocolVersion uint64
+type ProtocolFeature uint64
+
+type ProtocolInfo struct {
+	// Version is the supported protocol version.
+	Version ProtocolVersion
+	// Features are supported protocol features.
+	Features []ProtocolFeature
+}
+
+func (info ProtocolInfo) Copy() ProtocolInfo {
```

Comment:
Types ProtocolVersion, ProtocolFeature, ProtocolInfo and the method Copy() has no description, but it is a public API.

Reply:
Added